api/admin/routers/auth.js

```diff
@@ -0,0 +1,19 @@
+const { Router } = require('express');
+const passport = require('../passport');
+
+const onSuccess = (req, res) => {
+  const script = `
+    <script>
+      window.opener.postMessage("success", "http://localhost:3000")
```

Comment:
This will need to be replaced by an env var with the client url OR find another way to get the opener in a secure way.